CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# ChangeLog
+
+## 0.0.1
+
+- 
```

Comment:
maybe worth mentioning the 3 extensions that are available?

Reply:
done